site/pages/about-us/q-and-a-slice/topic/index.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React, { Component } from 'react';
+import styles from './style.css';
+
+export type TopicProps = {
+  slug: string,
+  question: string,
+  answer: string,
+}
+
+class Topic extends Component {
+  constructor(props: TopicProps) {
```

Comment:
I think it needs the babel `transform-class-properties` plugin https://github.com/redbadger/aimia-poc-web/blob/master/containers/data-provider/index.js#L4